### PR TITLE
Update TextArea font styles

### PR DESCRIPTION
**Pull request by mthadley**
This updates the font size and line height, as noted [here](https://www.pivotaltracker.com/story/show/162743330).

@NoRedInk/design How does this look?
image

### CODE REVIEW

**Comment by bendansby** [comment]
The label doodad looks off. Did the text get bigger or did the container get smaller?

**Comment by bendansby** [comment]
Here is staging 
image

**Comment by mthadley** [comment]
Just compared and it looks like a difference of `line-height` in the label. It's `20px` on staging, and `15px` in the styleguide for this repo. Investigating why...

**Comment by bendansby** [comment]
Looking at it now, I think I was mistaken in specifying a 30px line height. Could we do 27px instead? Apologies for the change!

**Comment by mthadley** [comment]
How's this?
screen shot 2019-01-17 at 5 47 55 pm

**Comment by bendansby** [comment]
Byootyful (on the line height)!

**Comment by mthadley** [comment]
@bendansby Okay, looks like we have a global line-height of `20px` for all labels declared in one of the monolith stylesheets. These screenshots are from the styleguide app in this repo which doesn't contain that stylesheet, so instead we are getting `15px` from `reset.css`. I think when this reaches staging, it should look fine.

**TLDR:** Staging should be fine, it's just my screenshots from the nri-ui repo that look weird.

**Comment by bendansby** [comment]
Oh our styling code base...

Thanks!

**Comment by bendansby** [comment]
It's used in guided writes and peer reviews. And we want to apply these new styles to all instances.

**Comment by mthadley** [comment]
@avh4 @bendansby I just double checked, and I only see `TextArea.writing` called in Peer Reviews and Guided Drafts. I think we should be good!

Thank you both for reviewing. 🙏
